.github/CHANGELOG.md

```diff
@@ -1,7 +1,57 @@
-# Release 0.18.0
+# Release 0.18.0 (current release)
 
 ### New features since last release
 
+* PennyLane-Lightning now provides a high-performance
+  [adjoint Jacobian](http://arxiv.org/abs/2009.02823) method for differentiating quantum circuits.
+  [(136)](https://github.com/PennyLaneAI/pennylane-lightning/pull/136)
+  
+  The adjoint method operates after a forward pass by iteratively applying inverse gates to scan
+  backwards through the circuit. The method is already available in PennyLane's
+  `default.qubit` device, but the version provided by `lightning.qubit` integrates with the C++
+  backend and is more performant, as shown in the plot below:
+
+  <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane-lightning/master/doc/_static/lightning_adjoint.png" width=70%/>
```

Comment:
Here it is (will only work when we merge into `master`):
https://raw.githubusercontent.com/PennyLaneAI/pennylane-lightning/v0.18.0_changelog/doc/_static/lightning_adjoint.png